pkg/filesystem/filesystem.go

```diff
@@ -38,7 +38,7 @@ type FileSystem interface {
 	Mount(ctx context.Context, path string, complete bool) (bool, error)
 
 	// Unmount the file system
-	Unmount(ctx context.Context, path string) (bool, error)
+	Unmount(ctx context.Context, path string, complete bool) (bool, error)
```

Comment:
Add the parameter that indicates whether we've already successfully unmounted to the API.